openverse_catalog/dags/providers/provider_api_scripts/time_delineated_provider_data_ingester.py

```diff
@@ -235,9 +250,9 @@ def get_should_continue(self, response_json) -> bool:
                 " been fetched. Consider reducing the ingestion interval."
             )
             if self.should_raise_error:
-                raise AirflowException(error_message)
+                raise Exception(error_message)
```

Comment:
It's necessary to change this to an `Exception`, because `AirflowException`s are specifically _not handled_ by our ingestion error handling logic (meaning they can never be skipped).

Reply:
@stacimc Is this feedback worth adding to the code?

```
# Must be an `Exception` instead of an `AirflowException` to allow skipping.
```

Or something like that.

Reply:
Or potentially even a custom, `Exception`-derived exception subclass?

Reply:
Our own error handling in the super class will [re-raise this](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py#L250) as an `IngestionError` already.

This isn't a problem with this specific exception so much as a general point about error handling in the ingester classes: we probably shouldn't be raising `AirflowException`s ourselves anyway.